mappings/1.13-pre6#1.13-pre7.tinydiff

```diff
@@ -44,9 +44,9 @@ c	net/minecraft/unmapped/C_1353256
 		p	4			p_4
 	m	()V	<clinit>		<clinit>
 c	net/minecraft/unmapped/C_1361861
-	m	(Lnet/minecraft/unmapped/C_1361861$C_6533070;)V	m_7249279		forEachFace
+	m	(Lnet/minecraft/unmapped/C_1361861$C_6533070;)V	m_7249279		forEachDirection
```

Comment:
I don't think this is an accurate change? It looks to me like the method iterates over every face of the shape, which could any direction multiple times.

Reply:
fair